docs/GettingStarted.md

```diff
@@ -150,14 +160,16 @@ On core-01
 
     docker run -e CALICO_IP=fd80:24e2:f998:72d6::1:1 --name workload-F -tid phusion/baseimage:0.9.16
```

Comment:
Can't we use ubuntu:14.04 instead of phusion? There is no good reason to use it...............